### PR TITLE
Tidy the footer

### PR DESCRIPTION
**Pull request by tsibley**
A bunch of small corrections and minor tidying in the footer. See commit messages for details.

Tested locally by building docs.nextstrain.org with this branch with:

```
env PYTHONPATH=$HOME/nextstrain/sphinx-theme/lib/ make html
```

### CODE REVIEW

**Comment by tsibley** [comment]
Using the RTD previews, you can also compare the footers [before](https://nextstrain-sphinx-theme--11.org.readthedocs.build/en/11/) and [after](https://nextstrain-sphinx-theme--10.org.readthedocs.build/en/10/).

**Review by eharkins — APPROVED** [review]
Thanks @tsibley!

Perhaps this is a separate issue related to the footer but it seems like the theme footer is out of sync with the nextstrain.org footer. E.g. myself and Moira and Cassia (and perhaps others) are not included in the "Nextstrain is built by" gallery of pictures/names.

**Comment by tsibley** [comment]
>  E.g. myself and Moira and Cassia (and perhaps others) are not included in the "Nextstrain is built by" gallery of pictures/names.

Oh! Well let's fix that too then!

**Comment by tsibley** [comment]
@eharkins [Look better](https://nextstrain-sphinx-theme--10.org.readthedocs.build/en/10/) now?

File: lib/nextstrain/sphinx/theme/footer.html [hunk 1]
```diff
@@ -22,182 +22,87 @@
...
+    {{ TeamMember(name="Barney Potter", image="barney-potter.jpg", link="http://bedford.io/team/barney-potter/") }},
+    {{ TeamMember(name="Pavel Sagulenko", image="pavel-sagulenko.jpg", link="https://neherlab.org/pavel-sagulenko.html") }},
+    {{ TeamMember(name="Charlton Callender", image="charlton-callender.jpg", link="http://bedford.io/team/charlton-callender/") }}
   </div>
 
 
   <div class="footer-small-text">
-    All <a href="https://github.com/nextstrain">source code</a> is freely available under the terms of the <a href="https://github.com/nextstrain/auspice/blob/master/LICENSE.txt">GNU Affero General Public License</a>. Screenshots may be used under a <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0 license</a> and attribution to nextstrain.org must be provided.
+    All <a href="https://github.com/nextstrain">source code for Nextstrain</a> is freely available under the terms of an <a href="https://opensource.org/osd">open-source license</a>, typically <a href="https://opensource.org/licenses/AGPL-3.0">AGPL-3.0</a> or <a href="https://opensource.org/licenses/MIT">MIT</a>.
```
Comment (jameshadfield) [inline lib/nextstrain/sphinx/theme/footer.html:65]: Which of our projects are MIT? This doesn't appear on the main (nextstrain.org) footer

**Review by jameshadfield — APPROVED** [review]
Apart from the MIQ question above, this looks good to me. Nice to have things back in sync.

**Comment by eharkins** [comment]
> @eharkins Look better now?

Looks great, thanks! I wonder how we can continue to keep the footer on nextstrain.org (and any other potential instances of this footer) in sync with the docs footer / vice versa, in terms of both the list of team members and the html/css code. Assuming the code lives here, maybe there's a nice way to install the theme in nextstrain.org and import it there?

File: lib/nextstrain/sphinx/theme/footer.html (hunk 1 above)
Comment (tsibley) [reply lib/nextstrain/sphinx/theme/footer.html:65]: ncov, ncov-ingest, nextclade, cli to name a few, plus some others.

The footer on nextstrain.org is also due for an update.

**Comment by tsibley** [comment]
In terms of keeping it in sync, I don't think there's a particularly nice technical solution. Easier and simpler to just keep doing it manually. We can make a checklist in comments of the places we maintain this footer, so that anyone updating one place can port their changes to the others too.

**Pull request merged**